ui/l10n/es.js

```diff
@@ -2159,66 +2159,66 @@ var dictionary = {
     "message.setup.physical.network.during.zone.creation.basic": "Cuando se esta agregando un zona básica, puedes configurar un red física que se corresponda a una NIC del hipervisor. La red transporta varios tipos de tráfico.<br/><br/>Puedes también <strong>arrastrar y soltar</strong> otro tipos de tráfico en la interfaz física.",
     "message.setup.successful": "La configuración del cloud finalizo satisfactoriamente.",
     "message.snapshot.schedule": "Puedes definir la planificación de instantáneas recurrentes seleccionando de entre las opciones inferiores disponibles y aplicando su política preferente",
-    "message.specifiy.tag.key.value": "Por favor especifique una etiqueta con clave y valor",
-    "message.specify.url": "Por favor especifique la URL",
-    "message.step.1.continue": "Por favor seleccione una plantilla o ISO para continuar",
-    "message.step.1.desc": "Por favor seleccione una plantilla para su nueva instancia virtual. También puede escoger  seleccionar una plantilla en blanco en la que instalar una imágen ISO.",
-    "message.step.2.continue": "Por favor seleccione una oferta de servicio para continuar",
-    "message.step.3.continue": "Por favor seleccione una oferta en disco para continuar",
-    "message.step.4.continue": "Por favor seleccione al menos una red para continuar",
+    "message.specifiy.tag.key.value": "Por favor, especifique una etiqueta con clave y valor",
+    "message.specify.url": "Por favor, especifique la URL",
+    "message.step.1.continue": "Por favor, seleccione una plantilla o ISO para continuar",
+    "message.step.1.desc": "Por favor, seleccione una plantilla para su nueva instancia virtual. También puede escoger  seleccionar una plantilla en blanco en la que instalar una imágen ISO.",
+    "message.step.2.continue": "Por favor, seleccione una oferta de servicio para continuar",
+    "message.step.3.continue": "Por favor, seleccione una oferta en disco para continuar",
+    "message.step.4.continue": "Por favor, seleccione al menos una red para continuar",
     "message.step.4.desc": "Por favor, seleccione la red primaria a la que estará conectada su instancia virtual",
     "message.storage.traffic": "Tráfico entre los recursos internos de CloudStack, incluyendo cualquier componente que se comunique con el servidor de gestión, tales como anfitriones y las Mvs de sistema de CloudStack. Por favor, configure el tráfico del almacenamiento aquí.",
-    "message.suspend.project": "¿Está seguro que desea suspender este proyecto?",
+    "message.suspend.project": "¿Está seguro de que desea suspender este proyecto?",
```

Comment:
I'm not a linguist but I think 'de' is not needed in this case, even though on speaking both ways may be accepted :)

Reply:
Mira el artículo de Wikipedia sobre [queísmo](https://es.wikipedia.org/wiki/Que%C3%ADsmo). Precisamente hay un ejemplo con «seguro de». En foros de idiomas y en la RAE vas a encontrar lo mismo.